superflore/generators/ebuild/ebuild.py

```diff
@@ -92,9 +98,8 @@ def get_ebuild_text(self, distributor, license_text):
         Generate the ebuild in text, given the distributor line
         and the license text.
 
```

Comment:
Please remove this blank line as well -- was there only to put a space between the comment and the TODO.